dm/syncer/syncer.go

```diff
@@ -3160,21 +3164,29 @@ func (s *Syncer) isClosed() bool {
 }
 
 // Close closes syncer.
-func (s *Syncer) Close() {
+func (s *Syncer) Close(graceful bool) {
 	s.Lock()
 	defer s.Unlock()
 
 	if s.isClosed() {
 		return
 	}
 
+	if !graceful {
+		s.runCancel()
+		s.tctx.L().Warn("syncer is closing without graceful")
+	}
 	s.stopSync()
 	s.closeDBs()
 
 	s.checkpoint.Close()
 
-	if err := s.schemaTracker.Close(); err != nil {
-		s.tctx.L().Error("fail to close schema tracker", log.ShortError(err))
+	if graceful {
```

Comment:
this is may cause confuse , but  i don't have better way. need reviewer suggestion